dashboard/app/config.go

```diff
@@ -234,6 +236,24 @@ type KernelRepo struct {
 	CC CCConfig
 	// This repository should not be polled for commits, e.g. because it's no longer active.
 	NoPoll bool
+	// LabelIntroduced is assigned to a bug if it was supposedly introduced
+	// in this particular tree (i.e. no other tree from CommitInflow has it).
+	LabelIntroduced string
```

Comment:
We call this tags everywhere else, let's call it it tag here as well.

Reply:
In the new version of https://github.com/google/syzkaller/pull/3813 it's everywhere called a label.